openreview/conference/templates/programchairWebfield.js

```diff
@@ -252,6 +255,24 @@ var getInvitations = function() {
   return Webfield.getAll('/invitations', { regex: WILDCARD_INVITATION, expired: true });
 }
 
+var getPcAssignmentTagInvitations = function() {
+  return Webfield.getAll('/invitations', { regex: PC_PAPER_TAG_INVITATION, tags: true});
+}
+
+var getPcAssignmentTags = function() {
+  Webfield.getAll('/tags', { invitation: PC_PAPER_TAG_INVITATION})
+  .then(function(results) {
+    if (results && results.length) {
+      results.forEach(function(tag){
+        if (! (tag.forum in pcTags)) {
+          pcTags[tag.forum] = [];
+        }
+        pcTags[tag.forum].push(tag);
```

Comment:
is there more than one tag per forum?

Reply:
No, there should be only one. I remembered that the tag widget in the bid web-field created multiple tags when one clicks quickly on the options for one paper. Although that does not apply here, I just wanted to account for that. Also, the value for pcTags[tag.forum] has to be an array as that is what the tagWidget expects.